spec/shared_examples/options.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This function is used to ensure that every config option
+# has a :type and :description key, as they are required for
+# generating the JSON schemas and validating config.
+def assert_type_description(definitions)
```

Comment:
This should be in `spec/lib` rather than `shared_examples`. This directory is specifically for [shared examples](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples).